README.md

```diff
@@ -1388,6 +1388,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [storyset.com](https://storyset.com/) — Create awesome free customised illustrations for your project using this tool.
   * [smartmockups.com](https://smartmockups.com/) — Create product mockups, 200 free mockups.
   * [tabler-icons.io](https://tabler-icons.io/) — Over 1500 free copy and paste SVG editable icons.
+  * [tailwindcss animated](https://www.tailwindcss-animated.com/) - Extended animation utilities for Tailwind CSS. Generate animation for tailwind using the configurator(https://www.tailwindcss-animated.com/configurator.html)
```

Comment:
This really is just a library, I dont mind things like icons and such - quite generally useful - but a tailwind specific library seems like there are better places to list it than a list for SaaS forware